src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -758,15 +836,21 @@ class ACCUpdateDirective(ACCStandaloneDirective):
                         clause on the update directive (this instructs the
                         directive to silently ignore any variables that are not
                         on the device).
+    :param async_queue: Make the directive asynchonous and attached to the \
```

Comment:
Probably not your fault but please could you fix the indentation of the continued lines L836-838 as they are also causing errors in the Ref. Guide.